packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.d.ts

```diff
@@ -11,7 +11,7 @@ export interface DropdownProps extends HTMLProps<HTMLDivElement> {
   onSelect(event: React.SyntheticEvent<HTMLDivElement>): void;
```

Comment:
Not needed now, but might be nice to make dropdownItems `ReactNode` instead of `ReactNode[]` the array variation is covered in `ReactNode`